### PR TITLE
fix: remove text brand element

### LINKED ISSUE #3306: Remove Branding title "Fuse Ignite"
## This is a...

<pre><code>
[ ] Feature request
[ ] Regression (a behavior that used to work and stopped working in a new release)
[X] Bug report  
[ ] Documentation issue or request
</code></pre>

Currently the branding header looks and the HTML title looks like

Instead the branding header should only be "Red Hat Fuse Online" (without Ignite) and the HTML title should also be only "Red Hat Fuse Online".

image

I can tweak this by adapting the ui-config and set `branding` to

```json
"branding": {
          "logoWhiteBg": "",
          "logoDarkBg": "",
          "iconWhiteBg": "assets/images/FuseOnlineLogo_Black.svg",
          "iconDarkBg": "assets/images/FuseOnlineLogo_White.svg",
          "appName": "",
          "favicon32": "/favicon-32x32.png",
          "favicon16": "/favicon-16x16.png",
          "touchIcon": "/apple-touch-icon.png",
          "productBuild": true
       }
```
This removes the "Ignite" in the branding but also leaves you with an empty HTML title.

image

### PR DESCRIPTION
**Pull request by gashcrumb**
fixes #3306

### CODE REVIEW

File: app/ui/src/app/app.component.html [hunk 1]
```diff
@@ -10,7 +10,6 @@
         alt="" />
       <img *ngIf="logoDarkBg" [class]="productBuild ? 'navbar-brand-name product' : 'navbar-brand-name'" [src]="loggedIn ? logoDarkBg : logoWhiteBg"
         alt="{{ appName }}" />
```
Comment (zregvart) [inline app/ui/src/app/app.component.html]: Should this `alt` also be changed to something else?

**Comment by zregvart** [comment]
I'm cool if @gashcrumb is cool...

File: app/ui/src/app/app.component.html (hunk 1 above)
Comment (gashcrumb) [reply app/ui/src/app/app.component.html]: no that should be fine

**Comment by gashcrumb** [comment]
@rhuss +1

**Pull request merged**